### DISCUSSION TITLE
Tabs and Dropdown

### DISCUSSION THREAD

**Discussion by alekseykurylev** [ariakit/ariakit#3319] [Questions] [answered]
Hello! How can I create a Tab component with dropdown items? So that when you click on one of the tabs, a drop-down list of tabs appears?

![image](https://github.com/ariakit/ariakit/assets/8526127/983c09fc-7e0b-4efc-a219-2949df505696)

**Comment by diegohaz** [comment, marked as answer]
There are a few experimental `CompositeOverflow` components currently under development for this purpose. They aren't yet included in the primary `@ariakit/react` package. If you're interested in testing them out: https://stackblitz.com/edit/jfjnjz?file=package.json,tab%2Findex.tsx&theme=dark

Feedback on the API and behavior will help us move toward a stable version of this component.

**Reply by alekseykurylev** [reply]
@diegohaz Are you planning to add width change support for this component? Example   https://github.com/mattrothenberg/react-overflow-list

**Reply by diegohaz** [reply]
Yes, I've got some preliminary code ready, but it will likely be an example at first, perhaps even using Matt's library.

Here's a Twitter thread about that: https://twitter.com/diegohaz/status/1472555722063941638
